app/scripts/metamask-controller.js

```diff
@@ -5704,6 +5562,7 @@ export default class MetamaskController extends EventEmitter {
         handleUpdate();
       },
       getStatePatches: () => patchStore.flushPendingPatches(),
+      ...this.controllerApi,
```

Comment:
Nit: Perhaps we could move this to before the patch methods, so they can't be accidentally overwritten? Not that that seems especially likely, but it's certainly not something we ever want to be possible.

Reply:
Done, good idea.